commit 8585878fe0fe1a4d7d0a4439ea017c3d28bf8a82
Author: Benjamin Peterson <benjamin@engflow.com>
Date:   Fri Nov 14 10:20:44 2025 -0800

    remove discovery gunk

diff --git a/google/cloud/compute/v1/BUILD.bazel b/google/cloud/compute/v1/BUILD.bazel
index 67f33d939..a0a837407 100644
--- a/google/cloud/compute/v1/BUILD.bazel
+++ b/google/cloud/compute/v1/BUILD.bazel
@@ -5,68 +5,6 @@ package(default_visibility = ["//visibility:public"])
 
 exports_files(glob(include = ["*grpc_service_config.json"]))
 
-##############################################################################
-# Discovery
-##############################################################################
-load(
-    "@com_google_disco_to_proto3_converter//rules_gapic:disco_to_proto.bzl",
-    "gapic_yaml_from_disco",
-    "grpc_service_config_from_disco",
-    "proto_from_disco",
-)
-
-_MESSAGE_IGNORE_LIST = [
-    "HttpHealthCheck",
-    "HttpsHealthCheck",
-    "HttpHealthCheckList",
-    "HttpsHealthCheckList",
-    "GetHttpHealthCheckRequest",
-    "GetHttpsHealthCheckRequest",
-    "PatchHttpHealthCheckRequest",
-    "PatchHttpsHealthCheckRequest",
-    "UpdateHttpHealthCheckRequest",
-    "UpdateHttpsHealthCheckRequest",
-    "InsertHttpHealthCheckRequest",
-    "InsertHttpsHealthCheckRequest",
-    "ListHttpHealthChecksRequest",
-    "ListHttpsHealthChecksRequest",
-    "DeleteHttpHealthCheckRequest",
-    "DeleteHttpsHealthCheckRequest",
-]
-
-_SERVICE_IGNORELIST = [
-    "HttpHealthChecks",
-    "HttpsHealthChecks",
-]
-
-proto_from_disco(
-    name = "compute_gen",
-    src = "compute.v1.json",
-    enums_as_strings = True,
-    input_config_path = "compute.config.json",
-    message_ignorelist = _MESSAGE_IGNORE_LIST,
-    previous_proto = "compute.proto",
-    service_ignorelist = _SERVICE_IGNORELIST,
-)
-
-grpc_service_config_from_disco(
-    name = "compute_grpc_service_config_gen",
-    src = "compute.v1.json",
-    input_config_path = "compute.config.json", 
-    message_ignorelist = _MESSAGE_IGNORE_LIST,
-    previous_proto = "compute.proto",
-    service_ignorelist = _SERVICE_IGNORELIST,
-)
-
-gapic_yaml_from_disco(
-    name = "compute_gapic_gen",
-    src = "compute.v1.json",
-    input_config_path = "compute.config.json",
-    message_ignorelist = _MESSAGE_IGNORE_LIST,
-    previous_proto = "compute.proto",
-    service_ignorelist = _SERVICE_IGNORELIST,
-)
-
 ##############################################################################
 # Common
 ##############################################################################
